yellowbrick/text/postag.py

```diff
@@ -73,6 +74,8 @@ class PosTagVisualizer(TextVisualizer):
     frequency: bool {True, False}, default: False
         If set to True, part-of-speech tags will be plotted according to frequency,
         from most to least frequent.
+    stack : bool {True, False}, default : False
+        If set to True, stacked barplot will be plotted.
```

Comment:
We might want to mention that `y` is required and what the stack will be - e.g. "Plot the PosTag frequency chart as a per-class stacked bar chart. Note that fit() requires `y` for this visualization." or something like that.